bpf/process/types/module.h

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: GPL-2.0
+/* Copyright Authors of Tetragon */
+
+#ifndef __LINUX_MODULE_H__
+#define __LINUX_MODULE_H__
+
+#ifndef TG_MODULE_NAME_LEN
+#define TG_MODULE_NAME_LEN 64
+#endif
+
+struct tg_load_module {
+	char name[TG_MODULE_NAME_LEN];
+	u32 sig_ok;
+	u32 pad;
+};
```

Comment:
this should have `__attribute__((packed))`

Reply:
done